install_config/configuring_authentication.adoc

```diff
@@ -1657,28 +1657,33 @@ Claims are read from the JWT `id_token` returned from the OpenID identity
 provider and, if specified, from the JSON returned by the `*UserInfo*` URL.
 
 At least one claim must be configured to use as the user's identity. The
-link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[standard identity claim] is `sub`.
+standard identity claim is `sub`.
 
 You can also indicate which claims to use as the user's preferred user name,
 display name, and email address. If multiple claims are specified, the first one
-with a non-empty value is used. The
-link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[standard claims] are:
+with a non-empty value is used. The standard claims are:
 
 [horizontal]
-`sub`:: The user identity.
-`preferred_username`:: The preferred user name when provisioning a user.
+`sub`:: Short for "subject identifier". The remote identity for the user at the
+issuer.
+`preferred_username`:: The preferred user name when provisioning a user. A
+shorthand name that the user wishes to be referred, such as `janedoe`. Typically
+a value that corresponding to the users login or username in the authentication
+system (username, email, etc).
```

Comment:
s/system (username, email, etc)./system, such as user name or email address.
(IBM Style says "no" to both parentheses and etc.)